util/config/gather-clang-sysroot-arguments

```diff
@@ -7,7 +7,8 @@ SYS_BASIC="$CHPL_HOME/runtime/include/sys_basic.h"
 
 SYSTEM_CLANG_COMMANDS=`clang -### $SYS_BASIC 2>&1`
 
-if [ $? -eq 0 ]; then
+#if [ $? -eq 0 ]; then
+if []; then
```

Comment:
i ran into an old clang reference problem when I was trying to build chapel with llvm 3.3, and I found the problem:
/export/home/hzhang86/chapel/chapel/third-party/llvm/install/linux64-gnu/bin/clang -resource-dir /usr/bin/../lib/clang/3.0 -o conftest -O3   conftest.c  >&5

therefore I manually edit this, I think you should be fine if no old clang installed